android/src/main/AndroidManifest.xml

```diff
@@ -1,3 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.tkporter.sendsms">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.tkporter.sendsms">
+
+    <uses-permission android:name="android.permission.SEND_SMS" />
```

Comment:
Would this still prompt users of apps that are not using the direct-send feature to use this permission?